Pipfile

```diff
@@ -6,9 +6,7 @@ name = "pypi"
 [packages]
 python-consul = "~= 1.1"
 pyopenssl = "~= 18.0"
-certbot = "==1.1.0"
-acme = "==1.1.0"
-mock = "==3.0.5"
+cryptography = "~= 2.9"
```

Comment:
@mtyaka, is there a reason why this is now an explicit dependency? I can't find any direct usage of this library at all.

Reply:
This is a transitive dependency, but without manually pinning it in the Pipfile, `pipenv install` will install the latest version which is not compatible with python 3.5.